contracts/Staking.sol

```diff
@@ -2,46 +2,56 @@ pragma solidity 0.6.2;
 
 import "./StorageManager.sol";
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
 
 /// @title staking
 /// @author see ERC900 proposal
 /// @author Rinke Hendriksen <rinke@iovlabs.org>
 /// @notice implements the ERC900 interface, with some small modifications. The contract also handles native tokens.
-contract Staking {
+contract Staking is Ownable {
 
     using SafeMath for uint256;
 
     StorageManager storageManager;
-    // amount of token staked per address
-    mapping(address => uint256) internal amountStaked;
-    // total amount of token staked per address
-    uint256 internal _totalStaked;
+    // amount of tokens per token staked per address [account -> (tokenAddress -> amount)]
+    mapping(address => mapping(address => uint256)) internal amountStaked;
```

Comment:
For convention I think we should use `_amountStaked` for the naming as it is `internal`